recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ requirements:
     - pip
     - pytest-runner
     - setuptools >=20.0
-    - wheel
+    - wheel !=0.37.1
```

Comment:
It's not correct. `!=0.37.1` means "_NOT 0.37.1_", but the upstream source says `wheel~=0.37.1`, in other words, "_AROUND 0.37.1_"  https://github.com/PyCQA/astroid/blob/14a0d657809ab5b42e7d416b686953508e6d8b91/pyproject.toml#L33.
We can use `wheel >=0.37.1` or simple `wheel`